### PR TITLE
Images and inputs

### PR DESCRIPTION
**Pull request by vijaytembugade**
Live preview of Images: https://vijaytembugade.github.io/XenonUI-Component-Library/Components/Images/image.html
Live preview for Inputs : https://vijaytembugade.github.io/XenonUI-Component-Library/Components/Inputs/inputs.html

Create responsive Images, can check with mobile view.
Inputs includes
- forms 
- input fields
- checkbox and radio buttons

### CODE REVIEW

File: Components/Inputs/inputs.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  margin: 0.25rem 0rem;
+  padding: 0.25rem;
+}
+
+.input-error {
+  border: 1px solid var(--danger-color);
+  outline: 1px solid var(--danger-color);
+}
+
+.validation-error {
```
Comment (manojasarsa) [inline Components/Inputs/inputs.css:65]: For Validation error, some bottom margin can be given to look more good.

**Review by manojasarsa — APPROVED** [review]
Looks clean and good, checked on mobile as well. 👍

File: Components/Inputs/inputs.css (hunk 1 above)
Comment (vijaytembugade) [reply Components/Inputs/inputs.css:65]: Thanks. I will add that

**Review by tanveertkd — APPROVED** [review]
Looks good!

**Comment by preethasri** [comment]
Looks neat and good 👍

File: Components/Inputs/inputs.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+}
+
+.form-group button {
+  width: 10rem;
+}
+
+input,
+textarea {
+  display: inline-block;
```
Comment (t007rushi) [inline Components/Inputs/inputs.css:24]: If one block of code used many times just make it utility class for easy user experience acc. To OOCSS paradigm

**Review by t007rushi — APPROVED** [review]
Everything looks good merge it.

**Review by MarkVed17 — COMMENTED** [review]
These inputs look so clean, Vijay.

File: Components/Inputs/inputs.css (hunk 2 above)
Comment (vijaytembugade) [reply Components/Inputs/inputs.css:24]: Okay rushi. I'll add them

**Pull request merged**